### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by MarcosFierro**

### CODE REVIEW

File: app/controllers/api/v1/comments_controller.rb [hunk 1]
```diff
@@ -74,4 +68,11 @@ def set_comment
   def comment_params
     params.require(:comment).permit(:user_id, :article_id, :body)
   end
+
+  def notificate_new_comment(comment)
+    return unless Article.find(comment.article_id).project_id
```
Comment (AlvaroSenorale) [inline app/controllers/api/v1/comments_controller.rb]:
```suggestion
    return unless Article.find(comment.article_id).project
```

File: app/controllers/api/v1/comments_controller.rb [hunk 2]
```diff
@@ -74,4 +68,11 @@ def set_comment
...
+
+    Project.find(Article.find(comment.article_id).project_id)
```
Comment (AlvaroSenorale) [inline app/controllers/api/v1/comments_controller.rb]:
```suggestion
    Article.find(comment.article_id).project
```

File: app/modules/subscription_observer.rb [hunk 3]
```diff
@@ -6,6 +6,7 @@ def self.included(base)
   end
 
   def new_notification(notification)
+    notification.user_id = id
```
Comment (AlvaroSenorale) [inline app/modules/subscription_observer.rb]: You don't need this, right? When you do:
```ruby
notifications << notification
```
you're already setting the foreign key

Comment (MarcosFierro) [reply app/modules/subscription_observer.rb]: Ok, didn't have that clear

**Pull request merged**